### ISSUE TITLE
Directus tries to recreate existing columns

### ISSUE THREAD

**Issue by 42656e** [directus/directus#2319]
# Bug Report

## Steps to Reproduce

1. Import existing tables in your database
2. Go to Collections in the Admin Interface
3. Click on 'Manage'
4. Click on the collection (still in the Admin Interface)
5. Click on an existing column to add an interface
6. Select e.g. a data type

## Expected Behavior
The data type and the other settings can be done correctly.

## Actual Behavior
You get the error "Trouble processing request. Try again after refreshing the page."

## Other Context & Screenshots
In the error log you can find the error:
`[2019-03-18 20:05:58] api[_].ERROR: PDOException: SQLSTATE[42S21]: Column already exists: 1060 Duplicate column name 'lastusage' in /opt/directus/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:379`
So it seems like Directus is trying to create a new column instead of editing the old one.

## Technical Details

- Device: Desktop
- OS:  Windows 10, Firefox
- Web Server: Nginx
- PHP Version: 7.2
- Database: 8.0
- Install Method: latest release as zip

**Comment by syslogic** [comment]
Despite there are no preferred naming conventions for column-names in MySQL, snake-case would work better than camel-case. To work around this, just don't attempt to create columns with names which would result in duplicate names; eg. by renaming the column once to a another name (eg. `last_usage`).

**Comment by 42656e** [comment]
Directus advertises to be used with existing databases. So it should not dictate a concrete naming-convention. Especially it should not do it without the knowledge of the user.

**Comment by rijkvanzanten** [comment]
I believe @shealavington's PR might've fixed this issue. It added a manage button for existing fields too, which should negate this issue.

**Comment by shealavington** [comment]
I would agree—I think the change I made may have fixed this, however it's untested in terms of this issue.

**Comment by benhaynes** [comment]
@42656e and @syslogic — can either of you confirm this is working as of the lastest release?

**Comment by EPMatt** [comment]
Still experiencing this issue in v8.0.0-rc.1. 

With columns containing upper case letters can't edit anything in the Schema and Options tabs of the Update Field interface. The error `Trouble processing request. Try again after refreshing the page.` is shown and a few lines containing the error mentioned in the opening report are added to the error log.

Moreover I noticed that directus still imports columns in lowercase format, and also table names are forced to lowercase. This could potentially create major issues when working with existing databases which use upper case letters in their schema.

**Comment by benhaynes** [comment]
@bjgajjar can you take a look at this?
